Directory.Build.props

```
@@ -6,8 +6,8 @@
     <NewtonsoftJsonVersion>12.0.3</NewtonsoftJsonVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <AwsSdkSnsVersion>3.5.0.9</AwsSdkSnsVersion>
-    <AwsSdkSqsVersion>3.5.0.9</AwsSdkSqsVersion>
+    <AwsSdkSnsVersion>3.5.0</AwsSdkSnsVersion>
+    <AwsSdkSqsVersion>3.5.0</AwsSdkSqsVersion>
```

Tests can use latest.

```suggestion
    <AwsSdkSnsVersion>3.5.0.9</AwsSdkSnsVersion>
    <AwsSdkSqsVersion>3.5.0.9</AwsSdkSqsVersion>
```